package.json

```diff
@@ -1,22 +1,22 @@
 {
-	"name": "vscode-vlang",
+	"name": "extension",
 	"displayName": "V",
 	"description": "V Language everywhere (syntax highlighting, formatter, linter, snippets) for Visual Studio Code.",
-	"publisher": "0x9ef",
+	"publisher": "vlang-vscode",
```

Comment:
It should be just `vscode` imo

Reply:
The publisher name is tied to the [VS Marketplace entity](https://marketplace.visualstudio.com/publishers/vlang-vscode)

Reply:
Ok then